libs/libsndfile/Makefile

```diff
@@ -27,8 +27,8 @@ define Package/libsndfile
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Library for reading/writing audio files
-  URL:=http://www.mega-nerd.com/libsndfile/
-  MAINTAINER:=Peter Wagner <tripolar@gmx.at>
+  URL:=https://libsndfile.github.io/libsndfile/
+  DEPENDS:=+lame-lib +libmpg123
```

Comment:
Why do we need these dependencies right now? In the commit description, it says enable something, but without any details, why we need it, what is the size difference, etc.

Reply:
You're right, commit description has been updated